docs/api/api.yml

```diff
@@ -271,6 +271,20 @@ paths:
       security:
         - BearerAuth: []
 
+  /deposit/{id}/agreement:
+    get:
+      tags:
+      - "deposit"
+      parameters:
+      - $ref: "#/components/parameters/DepositId"
+      responses:
+        200:
+          description: OK
+          content:
+            'application/pdf': {}
+        500:
```

Comment:
I'm missing 401 and 404 responses.

Reply:
I will merge this PR and add the 401 and 404 afterwards.